src/Model/Shopgate/Order.php

```diff
@@ -109,4 +107,33 @@ public function save()
     {
         $this->_getResource()->save($this);
     }
+
+    /**
+     * Get all shipments for the order
+     *
+     * @return array
+     */
+    public function getReportedShippingCollections()
+    {
+        $data = $this->getData('reported_shipping_collections');
+        $data = unserialize($data);
+        if (!$data) {
```

Comment:
can rewrite as shorthand `return data ?: []`. Although, double check what encoder class returns when it's empty.